compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -4152,7 +4246,46 @@ private void defineInvokableSymbolParams(BLangInvokableNode invokableNode, BInvo
             }
             paramSymbols.add(symbol);
         }
+        return paramSymbols;
+    }
 
+    public void defineInvokableTypeNode(BLangFunctionTypeNode functionTypeNode, long flags, SymbolEnv env) {
+        BInvokableTypeSymbol invokableTypeSymbol = (BInvokableTypeSymbol) functionTypeNode.symbol;
+        List<BVarSymbol> paramSymbols = defineParametersOfFunctionType(functionTypeNode, env);
+        invokableTypeSymbol.params = paramSymbols;
+
+        BType retType = null;
+        if (functionTypeNode.returnTypeNode != null) {
+            symResolver.resolveTypeNode(functionTypeNode.returnTypeNode, env);
```

Comment:
The returned value can be assigned to a variable and be used in L4260 and L4261.